### PR TITLE
[DO NOT MERGE] Place all frontend test reports in a single subdirectory

### PR DESCRIPTION
**Pull request by rsynek**
Jenkins jobs expect test results to be stored in
**/target/-reports/TEST-.xml

The same change goes into VRP as well:
https://github.com/kiegroup/optaweb-vehicle-routing/pull/229

### CODE REVIEW

File: employee-rostering-frontend/package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "cypress:run": "cypress run"
   },
   "jest-junit": {
-    "outputDirectory": "./target/surefire-reports",
```
Comment (Christopher-Chianelli) [inline employee-rostering-frontend/package.json]: Will Jenkins pick up on test results if they are not in surefire-reports?

Comment (Christopher-Chianelli) [reply employee-rostering-frontend/package.json]: Ah, just saw the above comment; nvm

**Comment by rsynek** [comment]
@Christopher-Chianelli please wait with the merge, the test result does not show up on Jenkins yet.

**Comment by Christopher-Chianelli** [comment]
@rsynek is this good to merge yet?

**Comment by rsynek** [comment]
@Christopher-Chianelli yes, it's good to go, please merge unless there are any concerns.

**Pull request merged**